code/game/objects/structures.dm

```diff
@@ -21,6 +21,12 @@
 		if(3.0)
 			return
 
+/obj/structure/proc/force_climb(atom/movable/A)

```

Comment:
named this force_climb instead of do_climb due to already having a do_climb proc used when climbing onto structures